app/(white_navbar)/explore/RetroPGF5/_component/DynamicCard.tsx

```diff
@@ -38,11 +38,11 @@ export default function DynamicCard({
   reward = 0,
 }: iDynamicCard) {
   const categoryElement = useMemo(
-    () => handleCategoryRound5(category),
+    () => handleCategoryRound5(category, 'text-xs'),
```

Comment:
Parsing `text-size` through handleCategory seems a bit off since we didn't do this for all last round. Are there any reason for this?